engine/apps/user_management/types.py

```diff
@@ -0,0 +1,20 @@
+import typing
+
+
+class AlertGroupTableColumn(typing.TypedDict):
+    id: str
+    name: str
+    type: str
+
+
+class AlertGroupTableColumns(typing.TypedDict):
+    visible: typing.List[AlertGroupTableColumn]
+    hidden: typing.List[AlertGroupTableColumn]
+    default: bool
```

Comment:
moved these from `apps/user_management/constants.py`.. I was adding `GoogleCalendarSettings` and it seemed more appropriate to have a separate types file (vs "constants")